gradle/gradle-mvn-push.gradle

```diff
@@ -10,22 +10,18 @@ publishing {
             name = "installLocally"
             url = "${rootProject.buildDir}/localMaven"
         }
-    }
-
-    /**
-     * Want to push to an internal repository for testing?
-     * Set the following properties in ~/.gradle/gradle.properties.
-     *
-     * internalUrl=YOUR_INTERNAL_URL
-     * internalUsername=YOUR_USERNAME
-     * internalPassword=YOUR_PASSWORD
-     */
-    def internalUrl = providers.gradleProperty("internalUrl")
-    if (internalUrl.isPresent()) {
+        /**
+         * Want to push to an internal repository for testing?
+         * Set the following properties in ~/.gradle/gradle.properties.
+         *
+         * internalUrl=YOUR_INTERNAL_URL
+         * internalUsername=YOUR_USERNAME
+         * internalPassword=YOUR_PASSWORD
+         */
         maven {
-            name = "internal"
-            url = uri(internalUrl)
-            credentials(PasswordCredentials)
+          name = "internal"
+          url = providers.gradleProperty("internalUrl").forUseAtConfigurationTime()
```

Comment:
`forUseAtConfigurationTime` is deprecated: https://docs.gradle.org/current/javadoc/org/gradle/api/provider/Provider.html#forUseAtConfigurationTime--